code/modules/events/spacevine.dm

```diff
@@ -308,15 +308,12 @@
 /datum/spacevine_mutation/woodening/on_grow(obj/effect/spacevine/holder)
 	if(holder.energy)
 		holder.density = 1
+	holder.maxhealth = 100
+	holder.health = holder.maxhealth
 
-/datum/spacevine_mutation/woodening/on_hit(obj/effect/spacevine/holder, mob/hitter, obj/item/I, expected_damage)
-	if(!expected_damage)
-		return
-	if(hitter)
-		if(I)
-			. = I.force * 2
-		else
-			. = 8
+/datum/spacevine_mutation/woodening/on_hit(obj/effect/spacevine/holder, mob/living/hitter, obj/item/I, expected_damage)
+	if(I.is_sharp())
```

Comment:
If it's not sharp but is hot, it probably won't take damage. It should take damage from hot objects, at least if it doesn't have fireproof.